tests/TestOne/PackageA/Makefile

```diff
@@ -0,0 +1,10 @@
+SHELL=/bin/bash
+
+build: node_modules
+	@$(MAKE) -f <(../../../.bin/esy build) PackageA
+
+node_modules:
+	sh ./test.sh
+
+clean:
+	rm -rf _build _install
```

Comment:
I think we want these to somehow be `cur__target_dir` and `cur__install`. The challenge seems to be that these variables are set by by the make file spit out by `esy build` command above. Maybe we create another command `esy build-env` that creates a makefile that *only* has sets the environment variables for a package build, but doesn't actually build anything. Then the `clean` step could be:
```
@$(MAKE) -f <(../../../.bin/esy build-env) PackageA && rm -rf $cur__target_dir cur_install
```